README.md

```diff
@@ -84,6 +87,20 @@ As you can see the variables are prefixed with `TRANSMISSION_`, the variable is
 PS: `TRANSMISSION_BIND_ADDRESS_IPV4` will be overridden to the IP assigned to your OpenVPN tunnel interface.
 This is to prevent leaking the host IP.
 
+#### Use docker env file
+It is recommended to use a docker env file where you can easily store all your env variables and maintain multiple configurations for different providers.
```

Comment:
How about: "You can also use a docker env file to easily store all your env variables and ..." ?